src/emulator/dataconnectEmulator.ts

```diff
@@ -139,16 +137,22 @@ export class DataConnectEmulator implements EmulatorInstance {
       "--logtostderr",
       "-v=2",
       "generate",
-      `--service_location=${args.locationId}`,
       `--config_dir=${args.configDir}`,
       `--connector_id=${args.connectorId}`,
     ];
     const res = childProcess.spawnSync(commandInfo.binary, cmd, { encoding: "utf-8" });
+
+    logger.debug(res.stderr);
     if (res.error) {
```

Comment:
I think we would want to remove this. `--logtostderr` will re-direct all logs to stderr. Could be negligence on my end.

Have we tested it?

Reply:
This does pipe all logs to stderr - however, it is the stderr of the childprocess, so it is not visible to the end user unless we do this.. This pipes that to the main logger at debug level, so that we can at least use it for debugging issues. 
We could alternatively log at info level if we want these to always be displayed, which looks like:
<img width="1433" alt="Screenshot 2024-06-24 at 6 28 19 PM" src="https://github.com/firebase/firebase-tools/assets/4635763/8f99fcbd-9012-49b7-8b9a-417fa4ac27f6">
Its a bit noisy, but not too bad overall.

Reply:
Oh, thanks for the test run!

The info logs here looks helpful~ I was worrying if `if (res.error)` would be true in case of any stderror. Doesn't seem to be that case.